api/src/components/link/service.ts

```diff
@@ -8,6 +9,20 @@ export const create = async (fromPublicationId: string, toPublicationId: string)
         }
     });
 
+    const latestVersionFrom = await client.prisma.publicationVersion.findFirst({
+        where: {
+            versionOf: fromPublicationId,
+            isLatestVersion: true
+        },
+        select: { id: true }
+    });
```

Comment:
```suggestion
const latestVersionFrom = await publicationVersionService.get(fromPublicationId, 'latest');
```

Reply:
I tried to use this initially but typescript didn't think the id field was available on the result

Reply:
It works... you may need an if statement in some cases to make TS happy but in this case there's an if statement below this
<img width="724" alt="image" src="https://github.com/JiscSD/octopus/assets/48569671/0e3f77f8-55fd-460f-b9e0-1b12eb0a267b">

Reply:
Yeah it works... I don't know what was different before. Now pushed